### PR TITLE
Add pyratlib

### PR DESCRIPTION
**Pull request by pyratlib**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pyrat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  home: https://github.com/pyratlib/pyrat
+  summary: PyRat is a user friendly library in python to analyze data from the DeepLabCut. Developed to help researchers unfamiliar with programming can perform animal behavior analysis more simpler.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - pyratlib
+    - tuliofalmeida
+    - abnr
```
Comment (chrisburr) [inline recipes/pyrat/meta.yaml:48]: @tuliofalmeida @abnr Are you happy to be listed as maintainers here?

Comment (tuliofalmeida) [reply recipes/pyrat/meta.yaml:48]: Yes @chrisburr. I totally agree.

Comment (abnr) [reply recipes/pyrat/meta.yaml:48]: Yes @chrisburr

Comment (pyratlib) [reply recipes/pyrat/meta.yaml:48]: Hi Chris, everything OK with PyRAT?

**Pull request merged**